xbmc/cores/VideoPlayer/VideoRenderers/RenderManager.cpp

```diff
@@ -1398,7 +1398,14 @@ bool CRenderManager::GetStats(int &lateframes, double &pts, int &queued, int &di
 
 void CRenderManager::CheckEnableClockSync()
 {
-  if (fabs(m_fps - g_graphicsContext.GetFPS()) < 0.01)
+  // refresh rate can be a multiple of video fps
+  double diff;
+  if (g_graphicsContext.GetFPS() >= m_fps)
+    diff = fmod(g_graphicsContext.GetFPS(), m_fps);
```

Comment:
Okay - I was assuming you wanted the 1001/1000 variants to be enabled - thinking the 0.01 was a 1% match, but it's actually more like a 0.01/30 = 0.03% match which excludes the 1001/1000 variants.
In that case the code does work for my example, so no problem.